src/routing.jl

```diff
@@ -116,13 +116,8 @@ end
 """
 Extract nodes ID's from route object
 """
-function get_route_nodes(m::MapData, route_indices::Array{Int64,1})
-    route_nodes = Array{Int}(undef,length(route_indices))
-    v = Dict{Int,Int}(reverse.(collect(m.v)))
-    for n = 1:length(route_nodes)
-        route_nodes[n] = v[route_indices[n]]
-    end
-    return route_nodes
+function get_route_nodes(m::MapData, route_indices::Vector{Int})
```

Comment:
Thanks for noticing this - this is some terrible leftover from code refactoring.
Let's change `Vector` to `AbstractVector` so the API is more type generic and follows Julia language recommendations (and of course such change is welcome in other places as well)